lib/babel-istanbul-instrumenter.js

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
Absolutely zero offense meant by this, but it seems as though your team may not have enough time available currently to move this project forward with pace.  This is a slight problem for us as I've essentially stopped maintenance of the blanket project in anticipation of this and our company paused all work on ember-cli-istanbul as well.

It may be prudent for us to resume work on ember-cli-istanbul at this stage as we have a minimum of 4 engineers who could contribute to the project immediately.  We would also be happy to grant collaborator status to members of the ember-cli team.

Let me know if this course would be overly controversial but I would rather have a useable, if slightly flawed, product that we iterate on than remain in limbo.